src/services/api/media.js

```diff
@@ -46,6 +46,8 @@ const media = {
     apiBase().delete(`${SITES}/${sitename}/${IMAGE_PATH}/${id}`).json(),
   deleteVideo: async ({ sitename, id }) =>
     apiBase().delete(`${SITES}/${sitename}/${VIDEO_PATH}/${id}`).json(),
+  updateAudio: async ({ id, sitename, data }) =>
```

Comment:
You might want to add `updateImage` and `updateVideo` apis in this PR as they are already being referenced in `useMedia`.